lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -372,6 +381,55 @@ newDBLayer trace fp timeInterpreter = do
                         <*> fromPersistValue retirementEpoch
             rights . fmap safeCast <$> rawSql query parameters
 
+        listPoolLifeCycleData epochNo =
+            rights . fmap parseRow <$> rawSql query parameters
```

Comment:
Same here, looks like we hide parse errors.

Reply:
See my comment here: https://github.com/input-output-hk/cardano-wallet/pull/2111#discussion_r484617141

Reply:
Fixed in 832ef1bc1df44bcd2b085590dfa93060fe797b1c.